h2o-core/src/main/java/water/parser/ParseSetup.java

```diff
@@ -27,6 +28,8 @@
   public static final int HAS_HEADER = 1;
   public static final int GUESS_COL_CNT = -1;
 
+  public BytesStats bytesStats;
```

Comment:
Sorry, it's not in master actually.
I'm going to use it everywhere, replacing the "width calculation" that is way too heavy and, as I have an example, faulty (have to investigate separately).